### ISSUE TITLE
Add a netdiff parser for OpenVPN that works on both of its status formats

### ISSUE THREAD

**Issue by rohithasrk** [ninuxorg/netdiff#43]
This aims to extract details from the output of different formats of OpenVPN. An example output can be shown as follows:

```
OpenVPN​ CLIENT LIST
Updated​ , ​ Mon​ ​ Mar​ ​ 27​ ​ 18​ : ​ 47​ : ​ 04​ ​ 2017
Common​ ​ Name​ , ​ Real​ ​ Address​ , ​ Bytes​ ​ Received​ , ​ Bytes​ ​ Sent​ , ​ Connected​ ​ Since
claud43nodo1​ , ​ 80.181​ . ​ 191.1​ : ​ 50268​ , ​ 53875636​ , ​ 192011412​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 09​ : ​ 06​ : ​ 20​ ​ 2017
LeMonacelle​ , ​ 2.226​ . ​ 154.66​ : ​ 44846​ , ​ 5164751​ , ​ 19264301​ , ​ Mon​ ​ Mar​ ​ 27​ ​ 16​ : ​ 10​ : ​ 59​ ​ 2017
Syskrack​ , ​ 79.26​ . ​ 49.2​ : ​ 60616​ , ​ 56928467​ , ​ 189355476​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 09​ : ​ 06​ : ​ 19​ ​ 2017
Kali2​ , ​ 2.226​ . ​ 154.66​ : ​ 55438​ , ​ 52681920​ , ​ 193489019​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 09​ : ​ 06​ : ​ 20​ ​ 2017
mirko​ , ​ 79.36​ . ​ 196.24​ : ​ 65039​ , ​ 1057697​ , ​ 245684199​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 09​ : ​ 06​ : ​ 20​ ​ 2017
buda​ , ​ 79.12​ . ​ 108.6​ : ​ 62026​ , ​ 1061815​ , ​ 245676448​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 09​ : ​ 06​ : ​ 23​ ​ 2017
pomezia​ , ​ 95.251​ . ​ 243.132​ : ​ 50275​ , ​ 1058494​ , ​ 245684089​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 09​ : ​ 06​ : ​ 21​ ​ 2017
ROUTING TABLE
Virtual​ ​ Address​ , ​ Common​ ​ Name​ , ​ Real​ ​ Address​ , ​ Last​ ​ Ref
aa​ : ​ f7​ : ​ ef​ : ​ 8f​ : ​ 55​ : ​ 52​ , ​ Syskrack​ , ​ 79.26​ . ​ 49.2​ : ​ 60616​ , ​ Mon​ ​ Mar​ ​ 27​ ​ 18​ : ​ 45​ : ​ 29​ ​ 2017
8a​ : ​ 0b​ : ​ ac​ : ​ 35​ : ​ 42​ : ​ c6​ , ​ LeMonacelle​ , ​ 2.226​ . ​ 154.66​ : ​ 44846​ , ​ Mon​ ​ Mar​ ​ 27​ ​ 16​ : ​ 11​ : ​ 01​ ​ 2017
a6​ : ​ 26​ : ​ 32​ : ​ 97​ : ​ 8b​ : ​ 6c​ , ​ claud43nodo1​ , ​ 80.181​ . ​ 191.1​ : ​ 50268​ , ​ Sun​ ​ Mar​ ​ 26​ ​ 23​ : ​ 14​ : ​ 10​ ​ 2017
c4​ : ​ 6e​ : ​ 1f​ : ​ ff​ : ​ 02​ : ​ 23​ , ​ Kali2​ , ​ 2.226​ . ​ 154.66​ : ​ 55438​ , ​ Mon​ ​ Mar​ ​ 27​ ​ 18​ : ​ 45​ : ​ 29​ ​ 2017
GLOBAL STATS
Max​ bcast​ / ​ mcast queue length​ , ​ 11
END
```

**Comment by nemesisdesign** [comment]
I forgot to mention there's a library which could help us in this task: https://github.com/tonyseek/openvpn-status

**Comment by nemesisdesign** [comment]
Merged (see #44 and #47 for reference).

**Comment by nemesisdesign** [comment]
PS: thanks @rohithasrk! 👍
